docs/business.yaml

```diff
@@ -1417,6 +1514,124 @@ components:
+            expireDateCurrExt:
+              type: string
+              format: date
+              description: Date of expiration for current extension, if applicable. Required when requesting an additional extension (when extReqForAgmYear is true).
+              example: '2024-04-10'
+            totalApprovedExt:
+              type: integer
+              description: Total duration of extension approved, measured in months.
+              example: 6
+            extensionDuration:
+              type: integer
+              description: Duration of the current extension request in months. Maximum is 6 months per request.
+              example: 6
+            expireDateApprovedExt:
+              type: string
+              format: date
+              description: Expiration date for approved extension.
+              example: '2024-07-10'
+            currentDate:
+              type: string
+              format: date
+              description: The current date at the time of the AGM extension request.
+              example: '2023-07-21'
+            alreadyExtended:
+              type: boolean
+              description: Indicates if the AGM has already been extended.
+              example: false
+            requestExpired:
+              type: boolean
+              description: Indicates if the request for extension has expired.
+              example: false
+            isGoodStanding:
+              type: boolean
+              description: Indicates if the company is in good standing.
+              example: true
+            incorporationDate:
+              type: string
+              format: date-time
+              description: The date and time when the company was incorporated.
+              example: '2024-06-20T17:31:58.000Z'
+            agmYear:
+              type: string
+              description: The year of the AGM for which the extension is being requested. Usually same as 'year'.
+              example: '2024'
+            prevAgmDate:
+              type: string
+              format: date
+              nullable: true
+              description: The date of the previous AGM, if applicable.
+              example: null
+            isPrevExtension:
+              type: boolean
+              description: Indicates if there was a previous extension for this AGM.
+              example: false
+            prevExpiryDate:
+              type: string
+              format: date
+              nullable: true
+              description: The expiry date of the previous extension, if applicable.
+              example: null
+            agmDueDate:
+              type: string
+              format: date
+              description: The due date for the AGM after the extension is applied.
+              example: '2026-06-20'
+            isEligible:
+              type: boolean
+              description: Indicates if the company is eligible for an AGM extension.
+              example: true
+          x-examples:
```

Comment:
Since we are providing comprehensive request and response examples under the path. Is the x-example necessary?

Reply:
Yes, we have provided comprehensive request and response examples under the path. The example under the schema was added following the existing writing style of other schemas in the codebase. If we think it is unnecessary, I can remove it.

Reply:
I didn't put x-examples under AGM location change schema. I think we had this in other schemas, since there weren't examples under each path.